lib/test.js

```diff
@@ -106,8 +107,11 @@ Test.prototype.test = function (name, opts, cb) {
     });
 };
 
-Test.prototype.comment = function (msg) {
+Test.prototype.comment = function () {
```

Comment:
i think it'd still be good to leave `msg` here tho, so the function's `.length` is correct :-)